packages/core/src/keys.ts

```diff
@@ -38,7 +38,29 @@ export namespace CoreBindings {
    */
   export const SERVERS = 'servers';
 
+  /**
+   * Binding tag for servers
+   */
+  export const SERVER_TAG = 'server';
```

Comment:
Instead of using `_TAG` suffix, can we create a new namespace instead?

```ts
CoreBindings.Tags.SERVER
```

Reply:
Or perhaps better: `CoreTags.SERVER`.

So far, namespaces like `CoreBindings`, `RestBindings`, etc. were holding binding keys (addresses) only.